components/Modal.vue

```diff
@@ -44,6 +54,12 @@ export default {
         this.$emit('close-modal')
       }
     })
+  },
+  methods: {
+    urlGitHub (path, extension) {
+      const url = 'https://github.com/eucp-project/data-catalogue/blob/main/content' + path + extension
```

Comment:
Was it a conscious decision to refer to  ..../blob/main/.... instead of .../edit/main/... ?

On the one hand, I think this is less "intimidating" than the real editing page. on the other hand, people might not immediately recognize the edit button. Or perhaps editing doesn't work if you're not logged in? 

What's your take on this?

Reply:
Good suggestion! I think that's much easier for the user to have the content immediately editable after clicking the "edit on github". I will change it.